cmd/lakefs/cmd/kv.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/kv"
+	"github.com/treeverse/lakefs/pkg/logging"
+	"github.com/treeverse/lakefs/pkg/version"
+)
+
+const (
+	ScanCmdMinArgs = 1
+	ScanCmdMaxArgs = 2
+	GetCmdNumArgs  = 2
+)
+
+var kvCmd = &cobra.Command{
+	Use:    "kv",
+	Short:  "Inspect lakeFS' Key-Value Store",
+	Hidden: true,
+}
+
+var kvGetCmd = &cobra.Command{
+	Use:    "get <partition key> <key>",
+	Short:  "Return the value for the given key under the given partition",
+	Hidden: true,
+	Args:   cobra.ExactArgs(GetCmdNumArgs),
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := logging.Default()
+		cfg := loadConfig()
+
+		ctx := cmd.Context()
+		logger.WithField("version", version.Version).Info("lakeFS kv")
+
+		kvParams := cfg.GetKVParams()
+		kvStore, err := kv.Open(ctx, kvParams)
+		if err != nil {
+			logger.WithError(err).Fatal("Failed to open KV store")
+		}
+		defer kvStore.Close()
+
+		val, err := kvStore.Get(ctx, []byte(args[0]), []byte(args[1]))
+		if err != nil {
+			logger.WithError(err).Fatal("Failed to get value")
+		}
+		prettyVal, err := kv.ToPrettyString(args[1], val.Value)
```

Comment:
If we named the format and fetch it here - without calling a function that will format it for us, it will enable us to dump and later use this information.

Reply:
`ToPrettyString` change to `ToKvObject` returning an object to support json. Output is later printed as json